tests/test_chi_ssa_50.py

```diff
@@ -0,0 +1,76 @@
+import json
+from datetime import datetime
+from os.path import dirname, join
+
+import pytest
+from city_scrapers_core.utils import file_response
+from freezegun import freeze_time
+
+from city_scrapers.spiders.chi_ssa_50 import ChiSsa50Spider
+
+test_response = file_response(
+    join(dirname(__file__), "files", "chi_ssa_50.html"),
+    url="http://southeastchgochamber.org/special-service-area-50/",
+)
+spider = ChiSsa50Spider()
+
+freezer = freeze_time("2019-10-27")
+freezer.start()
+
+parsed_items = [item for item in spider.parse(test_response)]
+
+freezer.stop()
+
+with open(join(dirname(__file__), "files", "chi_ssa_50.json")) as expected_file:
```

Comment:
The expected values should be listed in this test file and not in a separate file, so the JSON file can be removed and the expected values should be in this file

Reply:
Ok, I will remove the json file and make the change